### ISSUE TITLE
Publish to Maven Central, because JCenter is going away soon

### ISSUE THREAD

**Issue by ViliusSutkus89** [ViliusSutkus89/pdf2htmlEX-Android#23]
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by TomTasche** [comment]
I think I've suggested this before: https://docs.github.com/en/actions/guides/publishing-java-packages-with-maven#publishing-packages-to-github-packages

**Comment by ViliusSutkus89** [comment]
I experimented with GitHub maven packages. As usual with everything from Microsoft ... nothing works reliably.
Turns out that GitHub maven packages only keep the latest version that I publish.
Whatever version that is, a release or a -SNAPSHOT, which would be published on each successful build.
I'm pretty sure you don't want to depend on my -SNAPSHOT's.

Will see how it goes with Maven Central.
